### PR TITLE
[DA-2540]  Adding PubSub notifications for GHR3

### PR DESCRIPTION
**Pull request by dtharpeuno**
## Resolves *[ticket DA-2450]*
https://precisionmedicineinitiative.atlassian.net/browse/DA-2540

## Description of changes/additions
Editing configs based on notifications edited/added
Added stable to tool

## Tests
- [] unit tests

### CODE REVIEW

**Review by j-kanuch — APPROVED** [review]

File: rdr_service/config/pub_sub_config_sandbox.json [hunk 1]
```diff
@@ -3,64 +3,65 @@
...
-      "topic_project": "all-of-us-rdr-sandbox",
-      "payload_format": "JSON_API_V1",
-      "object_name_prefix": "Wgs_sample_raw_data/SS_VCF_research/",
       "event_types": [
         "OBJECT_FINALIZE"
-      ]
+      ],
+      "id": "58",
+      "object_name_prefix": "Wgs_sample_raw_data/ "
+      "SS_VCF_research/",
```
Comment (kenny-skaggs) [inline rdr_service/config/pub_sub_config_sandbox.json]: How does this line end up working? Or was this a typo?

Comment (dtharpeuno) [reply rdr_service/config/pub_sub_config_sandbox.json]: Good catch, fixed

Comment (kenny-skaggs) [reply rdr_service/config/pub_sub_config_sandbox.json]: I probably would have never seen this on my own, only noticed it because Codacy flagged it 😝

Comment (dtharpeuno) [reply rdr_service/config/pub_sub_config_sandbox.json]: Ah, was about to give you a cool nickname like 'Eagle Eye'

File: rdr_service/config/pub_sub_config_stable.json [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    {
+      "bucket": "genomics-raw-northwest",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "26",
+      "object_name_prefix": "W3SC_manifests/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
```
Comment (kenny-skaggs) [inline rdr_service/config/pub_sub_config_stable.json]: I'm not sure how the `topic_project` is used, but just wanted to confirm that the Stable JSON pub-sub config entries should have the topic_project set to "all-of-us-rdr-prod".

Comment (dtharpeuno) [reply rdr_service/config/pub_sub_config_stable.json]: Bad copypasta...

https://github.com/all-of-us/raw-data-repository/pull/2939/commits/0d7f89824fbd107554247bb3c6dd3fd7d4b01d77

**Review by kenny-skaggs — APPROVED** [review]

**Pull request merged**